integrity-backend/input-starling-capture-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,50 @@
+      "signature": [
+        {
+          "proofHash": "7b27933431f5eb258b8e4cea718219031d7a582108644e7fc9beee6a9055e47b",
+          "provider": "AndroidOpenSSL",
```

Comment:
C2PA action will switch between `AndroidOpenSSL` and `Zion` to create https://github.com/starlinglab/integrity-schema/blob/main/c2pa/claim-tool-examples/claim_create.json#L69-L102

Whether the Zion method uses "session" or not, we can tell from the pubkey. We'll assume both are part of the `numbers-Zion`.